op-program/client/l2/engine.go

```diff
@@ -43,7 +41,8 @@ func (o *OracleEngine) L2OutputRoot(l2ClaimBlockNum uint64) (eth.Bytes32, error)
 	if err != nil {
 		return eth.Bytes32{}, fmt.Errorf("failed to open L2 state db at block %s: %w", outBlock.Hash(), err)
 	}
-	withdrawalsTrie, err := stateDB.Database().OpenStorageTrie(outBlock.Root, predeploys.L2ToL1MessagePasserAddr, stateDB.GetStorageRoot(predeploys.L2ToL1MessagePasserAddr))
+	// TODO(Seb) confirm
+	withdrawalsTrie, err := stateDB.OpenStorageTrie(predeploys.L2ToL1MessagePasserAddr)
```

Comment:
Note for reviewers, added `OpenStorageTrie` in https://github.com/ethereum-optimism/op-geth/pull/215/commits/298900cf96512f9b4c3893e607b338112d62f7d1 (discussed with @ajsutton)

Reply:
Seems like other reviewers were happy with this.